### PR TITLE
Bah 781 event description

### PR DESCRIPTION
**Pull request by imeecuison**
## Description
Event Description pages for 781 and 781a.
http://localhost:3001/disability-benefits/apply/form-526-all-claims/disabilities/ptsd-secondaryEvent-description-0
http://localhost:3001/disability-benefits/apply/form-526-all-claims/disabilities/ptsd-event-description-0

## Testing done
tests/config/781.unit.spec.js

## Screenshots
screen shot 2018-11-20 at 9 50 20 am
screen shot 2018-11-20 at 9 50 01 am

## Acceptance criteria
- [ ] User is able to populate “Please tell us what happened during the event or situation. The information you give us here will help us research your claim. Provide the level of detail that you’re comfortable sharing. You don’t have to repeat any information that you’ve already shared.” - Free Text
- [ ] Field validations (if applicable per the linked document for Validations above) are enforced in the UI
- [ ] Error messages (if applicable per the linked document for Validations above) are displayed in the UI

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — COMMENTED** [review]
Just to be consistent can you rename files and paths from `event` to `incident`?  That would better match the other incident pages that are being added.

**Comment by imeecuison** [comment]
@dmccurdybah : I changed event to incident. Thank you!

**Review by devin-mccurdy — APPROVED** [review]

**Comment by imeecuison** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/disability-benefits/all-claims/config/form.js
Comment (rianfowler) [inline src/applications/disability-benefits/all-claims/config/form.js]: Can you help me understand why this is hard coded to three?

File: src/applications/disability-benefits/all-claims/utils.jsx
Comment (rianfowler) [inline src/applications/disability-benefits/all-claims/utils.jsx]: maybe assign a default value to `view:doneEnteringSecondaryIncidents` so "equals false" can be used.

File: src/applications/disability-benefits/all-claims/config/form.js
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/config/form.js]: @rianfowler : Change this to PTSD_INCIDENT_ITERATION in constants file. thank you!

**Review by rianfowler — APPROVED** [review]

**Comment by imeecuison** [comment]
@jbalboni or @cvalarida : Can I get a second review when you have a chance? Thank you!

**Review by jbalboni — APPROVED** [review]

**Pull request merged**
